rm_reporting/controllers/party_controller.py

```diff
@@ -28,10 +29,15 @@ def get_business_attributes(collection_exercise_id: str) -> dict[str, list]:
         f"ba.collection_exercise = '{collection_exercise_id}'"
     )
     logger.info("About to get party attributes", collection_exercise_id=collection_exercise_id)
-    with party_engine.begin() as conn:
-        attributes_result = conn.execute(text(attributes), {"collection_exercise_id": collection_exercise_id}).all()
-    result_dict = {str(getattr(item, "business_party_uuid")): item for item in attributes_result}
-    logger.info("Got party attributes", collection_exercise_id=collection_exercise_id)
+    try:
+        with party_engine.begin() as conn:
+            attributes_result = conn.execute(text(attributes), {"collection_exercise_id": collection_exercise_id}).all()
+        result_dict = {str(getattr(item, "business_party_uuid")): item for item in attributes_result}
+        logger.info("Got party attributes", collection_exercise_id=collection_exercise_id)
+    except SQLAlchemyError as e:
+        e.statement = ""
+        logger.error("Failed to get party attributes", collection_exercise_id=collection_exercise_id, error=e)
```

Comment:
You should be able to use e.code etc. instead of overriding the statement itself to get just the error information you want.

Reply:
The code no longer displays any information about the error, so this `statement` overriding is not relevant anymore and has been removed.